app/helpers/admins_helper.rb

```diff
@@ -0,0 +1,25 @@
+module AdminsHelper
+  
+  def super_admin?
+    @logged_in_user.role == 'super_admin'
```

Comment:
constants

AND don't user instance variables in helpers

AND devise providing method current_user